Cargo.lock

```diff
@@ -69,16 +69,13 @@ checksum = "baf1de4339761588bc0619e3cbc0120ee582ebb74b53b4efbf79117bd2da40fd"
 
 [[package]]
 name = "chrono"
-version = "0.4.22"
+version = "0.4.23"
```

Comment:
So we ask for version "0.4.2" but now we are using version "0.4.23"? Is that what this means? So we are using the latest version of chrono?

Reply:
Indeed, but it is not an important part of the PR. `cargo update` just happened to also update `chrono` to the latest version. The important information in this file is that `time` is removed, as it is a version that `cargo audit` reports as vulnerable.

Reply:
@bradjc This boils down to `Cargo.toml` dependency specifications by default assuming SemVer-compatible versioning schemes: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html

If we didn't have a `Cargo.lock` file, a user building `elf2tab` would automatically get the latest version of `chrono` within the constraints of `[0.4.2, 0.5]`. IMHO this is anything but intuitive (other tools require an explicit `~>` or `^`), and emphasizes the importance of the lock file.